.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+	"presets": ["es2015"]
```

Comment:
We have already  a definition of the preset in the gulp file itself  at https://github.com/ExLibrisGroup/primo-explore-devenv/blob/master/gulp/tasks/buildCustomJs.js#L27 
Did something specific not work with the current configuration?
Anyway it should not be in both places.